ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml

```diff
@@ -22226,6 +22226,17 @@ periodics:
     job-release: "4.13"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-openshift-logging-5.7-gcp-ipi-proxy-etcd-encryption-f7
+  reporter_config:
+    slack:
+      channel: '#aos-qe-logging'
```

Comment:
Are those slack notification changes for logging expected in this PR ?

Reply:
I believe no. Removing